app/controllers/admin/two_factor_authentications_controller.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+module Admin
+  class TwoFactorAuthenticationsController < BaseController
+    before_action :set_account
+
+    def destroy
+      @account.user.otp_required_for_login = false
+      @account.user.otp_backup_codes.clear
+      @account.user.save!
+      redirect_to admin_accounts_path
+    end
+
+    private
+
+    def set_account
+      @account = Account.find(params[:account_id])
```

Comment:
This could probably hit an `admin/users/:id/two_factor_authentication#destroy` route, since it's only doing things to the user record and not the account record.